pages/404.js

```diff
@@ -1,7 +1,47 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import Header from '../components/Header';
+
 export default function FourZeroFour() {
 	return (
-		<div className='404--wrapper'>
-			<h1 className='404--oops'>OOPS!</h1>
+		<div className='fourzerofour--wrapper'>
+			<Header />
+			<div className='fourzerofour--title-wrapper'>
+				<h1 className='fourzerofour--title'>OOPS!</h1>
+			</div>
+			<ul className='fourzerofour--ul'>
+				<li>
+					<Link href='/'>
+						<a className='fourzerofour--li-home'>Home</a>
```

Comment:
Close but there's a more optimal solution. `fourzerofour--li fourzerfour--li__with-hyphen` feel free to rename that but you get the idea.